daemon/cmd/daemon_main.go

```diff
@@ -575,6 +575,26 @@ func init() {
 	flags.String(option.IPMasqAgentConfigPath, "/etc/config/ip-masq-agent", "ip-masq-agent configuration file path")
 	option.BindEnv(option.IPMasqAgentConfigPath)
 
+	flags.StringSlice(option.IPAMClusterPoolV4CIDR, []string{}, fmt.Sprintf("IPv4 CIDR Range for Pods in cluster. Requires '%s=%s' and '%s=%s'",
+		option.IPAM, ipamOption.IPAMOperator,
+		option.EnableIPv4Name, "true"))
+	option.BindEnv(option.IPAMClusterPoolV4CIDR)
+
+	flags.StringSlice(option.IPAMClusterPoolV6CIDR, []string{}, fmt.Sprintf("IPv6 CIDR Range for Pods in cluster. Requires '%s=%s' and '%s=%s'",
+		option.IPAM, ipamOption.IPAMOperator,
+		option.EnableIPv6Name, "true"))
+	option.BindEnv(option.IPAMClusterPoolV6CIDR)
```

Comment:
These options should be marked as Hidden to not confuse the user.

Reply:
Done.